nix/kexec-installer/kexec-run.sh

```diff
@@ -84,4 +84,4 @@ else
 fi
 # We will kexec in background so we can cleanly finish the script before the hosts go down.
 # This makes integration with tools like terraform easier.
-nohup sh -c "sleep 6 && '$SCRIPT_DIR/kexec' -e" &
+nohup sh -c "sleep 6 && '$SCRIPT_DIR/kexec' -e --no-sync" &
```

Comment:
Also most people probably use this to replace the old system,
but in case someone just needs this to perform some filesystem maintenance,
we shouldn't uncleanly unmount the filesystem.

I thought about having a timeout:

```suggestion
nohup sh -c "sleep 6 && (timeout 30 sync; '$SCRIPT_DIR/kexec' -e --no-sync)" &
```

However this also is not ideal because unlike the kexec version this actually performs the sync before all userspaces processes are stopped.

So I think this should be a flag instead.

Reply:
I think we could have a way to allow to pass flags to kexec in general instead of just having a `--no-sync` flag.

Reply:
I actually also think now that this would be better. Allowing to pass in flags makes it flexible for all sorts of corner cases. Will try to work this out and push an update into this PR.